features/feature_files/1-common/FS1-5-consent-validation.feature

```diff
@@ -28,12 +28,12 @@ Feature: Validation on the fields required for consent from the bank to be obtai
         Then the service displays the following error message
             | sort Code-error | Enter a valid "Sort code" |
 
-    Scenario: Case Worker enters invalid Sort Code - not numbers 0-9
```

Comment:
It looks like you might've replaced an existing test instead of adding a new one. Perhaps we should re-add the "don't allow letters in sort codes" test?